docs/usage/sat_bootprep.md

```diff
@@ -526,6 +526,8 @@ for the COS product in VCS are stored in a branch named
 `sat bootprep` looks for a branch named `integration-2.4.99` from which to
 create CFS configuration layers.
 
+<!--- Lines 531 through 547 need to be updated with a different example since cne-install is going away. Do you have any suggestions? --->
```

Comment:
@haasken-hpe, @ekamin-hpe, or @jack-stanek-hpe, do you have any suggestions on how to change lines 531 through 547?

Reply:
Honestly my vote is to remove them, then start line 549 differently, something like this:

```
You can create VCS working branches that are not the default bootprep input file
branch names. For example, to use the branch `integration` for COS instead of
`integration-{{cos.version}}`, edit the bootprep input file so it reads as
follows.
```

Reply:
Sounds good, I'll update in my next commit.